src/av2/datasets/motion_forecasting/eval/submission.py

```diff
@@ -0,0 +1,102 @@
+# <Copyright 2022, Argo AI, LLC. Released under the MIT license.>
+
+"""Classes and utilities used to build submissions for the AV2 motion forecasting challenge."""
+
+from __future__ import annotations
+
+from collections import defaultdict
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, Final, List, Tuple
+
+import numpy as np
+import pandas as pd
+
+from av2.utils.typing import NDArrayNumber
+
+TrackPredictions = NDArrayNumber  # Track predictions are expected to be of shape (*, NUM_PRED_TIMESTEPS, 2)
+ScenarioPredictions = Dict[str, TrackPredictions]  # Mapping from track ID to track predictions
+PredictionRow = Tuple[str, str, int, NDArrayNumber, NDArrayNumber]
+
+SUBMISSION_COL_NAMES: Final[List[str]] = [
+    "scenario_id",
+    "track_id",
+    "prediction_rank",
```

Comment:
Does this mean we don't expect probabilities?

Reply:
Updated to add probabilities as an input.